$NetBSD: patch-data_Makefile.am,v 1.1 2018/06/11 23:55:50 youri Exp $

Use the pkgsrc specific solution to handle /etc files with the CONF_FILES
framework.

--- data/Makefile.am.orig	2014-11-04 03:46:00.000000000 +0000
+++ data/Makefile.am
@@ -4,7 +4,7 @@ dbusif_DATA = \
 	org.freedesktop.Accounts.xml 		\
 	org.freedesktop.Accounts.User.xml
 
-dbusconfdir   = $(sysconfdir)/dbus-1/system.d
+dbusconfdir   = ${EGDIR}/dbus-1/system.d
 dbusconf_DATA = org.freedesktop.Accounts.conf
 
 servicedir       = $(datadir)/dbus-1/system-services
